### ISSUE TITLE
add on vanished and cannot reinstall

### ISSUE THREAD

**Issue by pukkafukka** [mrknow/filmkodi#76]
The add on crashed and vanished now when i go to install it again it fails. For some reason it displays specto - ... Fork rather than specto - genesis fork, hangs on 3 percent for a while then fails.

**Comment by mrknow** [comment]
Right now we have problem wit server performance. Try install add-on tomorow

**Comment by blackcell1** [comment]
Yeah same issue, the addons uninstalled its self and now its hanging on a 3% install. I guess I'll wait until tomorrow.

**Comment by pukkafukka** [comment]
thank you

**Comment by mrknow** [comment]
Try install  update manualy 
[plugin.video.specto.zip](https://github.com/mrknow/filmkodi/files/120518/plugin.video.specto.zip)

**Comment by davie1888** [comment]
Worked using manual update mrknow :) Could you please sort the TV section download so I can chose where it goes to. TIA

**Comment by johndeere69** [comment]
great work, now you should change all the icons and make it yours

**Comment by kev7414** [comment]
Just checking on issues with server. Still not able to download through zip or super repo.

**Comment by karnage11** [comment]
yeah tried to install the Filmkodi repo and trying to get into it , it says Can't connect to repo
